rocAL/rocAL/source/bounding_box_graph.cpp

```diff
@@ -46,19 +46,19 @@ void BoundingBoxGraph::update_meta_data(MetaDataBatch *input_meta_data, decoded_
         memcpy(coords_buf.data(), input_meta_data->get_bb_cords_batch()[i].data(), input_meta_data->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
         BoundingBoxCords bb_coords;
         BoundingBoxCord temp_box;
-        temp_box.x = temp_box.y = temp_box.w = temp_box.h = 0;
+        temp_box.l = temp_box.t = temp_box.r = temp_box.b = 0;
         BoundingBoxLabels bb_labels;
         int m = 0;
         for (uint j = 0; j < bb_count; j++)
         {
             BoundingBoxCord box;
-            float temp_x, temp_y;
-            temp_x = (coords_buf[m++] * _dst_to_src_width_ratio);
-            temp_y = (coords_buf[m++] * _dst_to_src_height_ratio);
-            box.x = (temp_x > 0) ? temp_x : 0;
-            box.y = (temp_y > 0) ? temp_y : 0;
-            box.w = (coords_buf[m++] * _dst_to_src_width_ratio);
-            box.h = (coords_buf[m++] * _dst_to_src_height_ratio);
+            float temp_l, temp_t;
+            temp_l = (coords_buf[m++] * _dst_to_src_width_ratio);
+            temp_t = (coords_buf[m++] * _dst_to_src_height_ratio);
+            box.l = (temp_l > 0) ? temp_l : 0;
+            box.t = (temp_t > 0) ? temp_t : 0;
+            box.r = (coords_buf[m++] * _dst_to_src_width_ratio);
+            box.b = (coords_buf[m++] * _dst_to_src_height_ratio);
             bb_coords.push_back(box);
             bb_labels.push_back(labels_buf[j]);
```

Comment:
Labels are not changed here. So do we need to push them again?

Reply:
No, we dont need to push them again, have made the necessary change